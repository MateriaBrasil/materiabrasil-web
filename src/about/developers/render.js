import React from 'react'

import Colors from '../../Colors'
import Session from '../Session'
import Members from '../Members'
import Title from '../Title'
import Subtitle from '../Subtitle'
import Paragraph from '../Paragraph'
import developersList from './developersList'

export default () => (
  <Session id="developers">
    <Title color>Desenvolvedores</Title>
    <Paragraph color>
      Para o desenvolvimento desses direcionadores a MateriaMundi contou com uma
      equipe de consultores especialistas em diversas áreas.
    </Paragraph>
    <Subtitle>Conheça o nosso time!</Subtitle>
    <Members list={developersList} color={Colors.black} />
  </Session>
)
