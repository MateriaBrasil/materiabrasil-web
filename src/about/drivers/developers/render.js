import React, { Fragment } from 'react'

import Members from '../../Members'
import Subtitle from '../../Subtitle'
import Paragraph from '../../Paragraph'
import membersList from './developersList'

export default () => {
  return (
    <Fragment>
      <Subtitle>Desenvolvedores</Subtitle>
      <Paragraph>
        Para o desenvolvimento desses direcionadores a MateriaMundi contou com
        uma equipe de consultores especialistas em diversas áreas.
      </Paragraph>
      <Paragraph>Conheça o nosso time!</Paragraph>
      <Members membersList={membersList} />
    </Fragment>
  )
}
