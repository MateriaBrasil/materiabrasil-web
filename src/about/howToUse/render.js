import React from 'react'

import Section from '../Section'
import Title from '../Title'
import Video from '../../Video'
import VideoStyle from './VideoStyle'

export default () => (
  <Section
    id="how_to_use"
    backgroundImage="url('https://s3.amazonaws.com/materiamundi-us/static/backgrounds/material-3.jpg')"
  >
    <Title>Como usar</Title>
    <VideoStyle>
      <Video />
    </VideoStyle>
  </Section>
)
