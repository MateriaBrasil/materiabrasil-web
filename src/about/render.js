import React, { Fragment } from 'react'
import Typography from '@material-ui/core/Typography'

import Grid from './Grid'
import NavBar from '../navBar/NavBar'

export default props => routeProps => (
  <Fragment>
    <NavBar {...props} isTransparent="true" />

    <Grid>
      <Typography
        variant="display2"
        color="inherit"
        className="text"
        style={{
          color: '#fff',
          marginTop: 70,
          display: 'inline-block',
          fontWeight: 300,
        }}
      >
        <h2>
          <strong>Objetivo</strong>
        </h2>
        O objetivo da MateriaMundi oferecer acesso à informação de qualidade e
        parâmetros para que seus usuários possam escolher alternativas
        comercialmente disponíveis e socioambiental responsáveis para os
        desafios do dia a dia, variando desde o que vestimos até como
        construímos nossas casas.
        <br />
        Através da criação de uma comunidade ativa a plataforma permite a
        criação de conhecimento compartilhado sobre ações práticas para uma
        forma de projetar, produzir e consumir mais responsável com a sociedade
        e o meio ambiente.
        <h2>
          <strong>Como usar</strong>
        </h2>
        <h2>
          <strong>Equipe</strong>
        </h2>
        <h2>
          <strong>Board</strong>
        </h2>
        <h2>
          <strong>Direcionadores</strong>
        </h2>
      </Typography>
    </Grid>
  </Fragment>
)
