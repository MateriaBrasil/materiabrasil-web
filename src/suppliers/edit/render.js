import React from 'react';

import Form from '../form/Form';
import updateSelectedParams from './updateSelectedParams';

export default props => ({ info, update, updating, error }) => {
  return (
    <Form
      onSubmit={updateSelectedParams(update)}
      submitting={updating}
      updateError={error}
      initialValues={info}
      buttonText="Atualizar fornecedor"
      {...props}
    />
  );
};
