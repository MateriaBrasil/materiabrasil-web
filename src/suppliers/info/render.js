import React from 'react'
import Helmet from '../../Helmet'

import Info from './Info'

export default props => (info, infoProps) => {
  return (
    <Helmet
      title={infoProps.info.name}
      ogImage={infoProps.info.imageUrl}
    >
      <Info current={info} {...props} {...infoProps} />
    </Helmet>
  )
}
