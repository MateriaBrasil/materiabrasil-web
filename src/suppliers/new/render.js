import React from 'react';

import Form from '../form/Form';

export default props => ({ create, creating, error }) => {
  const handleSubmit = async function(e) {
    const response = {
      name: e.name,
      website: e.website,
      description: e.description,
      email: e.email,
      cnpj: e.cnpj,
      company_name: e.companyName,
      municipal_subscription: e.municipalSubscription,
      state_subscription: e.stateSubscription,
      phone: e.phone,
      company_revenue: e.companyRevenue,
      reach: e.reach,
      addresses_attributes: [
        {
          state: e.addresses.state,
          city: e.addresses.city,
          address_type: 'postal',
          street_address: e.addresses.streetAddress,
          country: e.addresses.country,
          zip_code: e.addresses.zipCode,
          phone_number: e.addresses.phoneNumber,
        },
      ],
    };

    create({
      supplier: response,
    });
  };

  return (
    <Form
      onSubmit={handleSubmit}
      submitting={creating}
      createError={error}
      buttonText="Registrar fornecedor"
      {...props}
    />
  );
};
