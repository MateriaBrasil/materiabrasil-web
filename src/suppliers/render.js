import React from 'react';
import Router from './Router';
import NaveBarAndContent from '../NavBarAndContent';
import Grid from '@material-ui/core/Grid';

export default props => routeProps => (
  <Grid style={{ backgroundColor: '#fff' }}>
    {/* <NaveBarAndContent {...props}> */}
    <Router {...props} />
    {/* </NaveBarAndContent> */}
  </Grid>
);
