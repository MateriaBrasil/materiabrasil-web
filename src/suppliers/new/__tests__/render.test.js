import React from 'react'
import renderer from 'react-test-renderer'

import render from '../render'

jest.mock('../Form', () => props => <div {...props}>Form</div>)

it('renders correctly', () => {
  const props = { foo: 'bar' }
  const createProps = {
    create: jest.fn(),
    creating: jest.fn(),
    error: jest.fn(),
  }
  const tree = renderer.create(render(props)(createProps)).toJSON()
  expect(tree).toMatchSnapshot()
})
