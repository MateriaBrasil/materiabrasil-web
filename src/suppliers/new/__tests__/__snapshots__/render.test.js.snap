// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  createError={[Function]}
  foo="bar"
  onSubmit={[Function]}
  submitting={[Function]}
>
  Form
</div>
`;
